pkgs/development/python-modules/distributed/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytest
+, pytest-repeat
+, pytest-faulthandler
+, pytest-timeout
+, mock
+, joblib
+, click
+, cloudpickle
+, dask
+, msgpack
+, psutil
+, six
+, sortedcontainers
+, tblib
+, toolz
+, tornado
+, zict
+, pyyaml
+, pythonOlder
+, futures
+, singledispatch
+}:
+
+buildPythonPackage rec {
+  pname = "distributed";
+  version = "1.22.1";
```

Comment:
There's a newer version per the one on [master](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/distributed/default.nix).

Reply:
Can you point out the specific commits to use? Otherwise the purpose of this is to solve the bug, not update to latest.

Reply:
You're already bumping from `1.15.1` to `1.22.1`. Sure, it's 816cc6dc8384af52f050090be2529c7ffaf44145.

Reply:
That could be a separate PR?